packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -19,16 +17,28 @@ export interface BaseInputOptionProps {
   disabled?: boolean;
 
   /**
-   * Defines the currently highlighted option element for keyboard navigation.
-   * Not to be confused with `selected`, which identifies the currently selected option
+   * Defines the currently highlighted option,
+   * and applies the relevant styles and aria attributes.
+   *
+   * Functionally similar to `:focus` state, however highlight not always implemented with true browser focus state
+   * (e.g. some components maintain the browser focus on the trigger element,
+   * and identify the "highlighted" option with the `aria-selected` attribute).
+   *
+   * Not to be confused with `checked`, which identifies the currently active/selected option.
    * @default false
    */
   highlighted?: boolean;
 
   /**
-   * Whether the component is selected, regardless of keyboard navigation
+   * Defines the currently selected/active element, regardless of interaction state.
+   *
+   * Functionally similar to a checkbox/radio's `checked` attribute,
+   * this identifies an option as currently selected.
+   *
+   * Note: There are no styling changes applied by this prop.
+   * `Checked` styles must be applied by the implementing component
    */
-  selected?: boolean;
+  checked?: boolean;
```

Comment:
Does this change not require us to update other components (we could test code mods here ! )

Reply:
It doesn't seem like these props have been used anywhere yet